Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.cs

```diff
@@ -23,7 +23,6 @@ namespace Microsoft.Toolkit.Uwp.UI.Animations
     /// </summary>
     public static partial class AnimationExtensions
     {
-        private static readonly EasingFunctionBase _defaultStoryboardEasingFunction = new CubicEase();
```

Comment:
I think that it would be nice but that would maybe decrease the ease of use of the toolkit. In the current form the user doesn't have to know the internal things like EasingFunctionBase and so on.